.github/workflows/depsreview.yaml

```diff
@@ -1,12 +1,13 @@
 name: 'Dependency Review'
 on: [pull_request]
 
-permissions:
```

Comment:
If we leave this action level permission here, does it apply to any jobs that omit the permissions configuration? If so, it seems safer to leave it in so that any future jobs that neglect to configure permissions will only get read (assuming the org level default is higher).

Reply:
contents: read

is the default anyhow. 

I just like to define it explicitly so it is clear what permissions a job has.

See the link in the pr description.